consensus/qbft/src/integration-test/java/org/hyperledger/besu/consensus/qbft/support/TestContextBuilder.java

```diff
@@ -398,7 +425,14 @@ private static ControllerAndState createControllerAndFinalState(
             : Collections.emptyMap();
     final QbftConfigOptions qbftConfigOptions = createGenesisConfig(useValidatorContract);
 
-    genesisConfigOptions.byzantiumBlock(0);
+    if (useLondonMilestone) {
+      genesisConfigOptions.londonBlock(0);
+    } else {
+      genesisConfigOptions.berlinBlock(0);
+    }
```

Comment:
Might be simpler to just use the MainnetProtocolSchedule with the genesis file instead. I think this and the above comment about zero base fees come down to how we want to write these tests. Would we rather create custom genesis files or use a generic genesis file with config in the tests?

Reply:
I'm not sure I follow exactly, but refactoring how we load the genesis is out of the scope of this PR I reckon.

Reply:
I think it would be more straight forward to load a full genesis file rather than just the genesis block....but yes we'd end up with lots of genesis files.

Reply:
I think prefer less genesis files to deal with so I think this approach is better after all.